### PR TITLE
Update services SQL

### LINKED ISSUE #3202: Last Check on Services - Time is not updated
Good Afternoon,

in Services Tab, the last check is not updated. It remains on the value when the service was first checked.

### PR DESCRIPTION
**Pull request by f0o**
Fixes #3202 

array_merge fails if the 2nd parameter is being unset before

### CODE REVIEW

**Comment by f0o** [comment]
Ready for merge, works on CI

File: check-services.php [hunk 1]
```diff
@@ -48,13 +48,9 @@
         if ($service_status != $status) {
             $update['service_changed'] = time();
         }
-        else {
-            unset($updated);
```
Comment (murrant) [inline check-services.php]: @f0o What was this even doing?  Did the $updated variable exist?

Comment (f0o) [inline check-services.php]: I think that else was a relict anyways. figured the main issue here was the webui part that pulled the wrong var.

File: html/includes/print-service.inc.php [hunk 2]
```diff
@@ -70,7 +70,7 @@
            <span class='box-desc'>$desc</span>
          </td>
          <td>
-           <span class='box-desc'>".date('r',$service['service_changed'])."</span>
+           <span class='box-desc'>".date('r',$service['service_checked'])."</span>
```
Comment (murrant) [inline html/includes/print-service.inc.php]: Shouldn't this just be `<span class='box-desc'>$checked</span>`?

**Pull request merged**